src/roadmap/roadmap.md

```diff
@@ -1,24 +1,110 @@
 
 Given the project's rapid [Release Schedule](releases.md), roadmap
 items are designed to provide a sense of direction over many releases.
 
-## Deis v2
+## Deployments
 
-A complete roadmap for post 2.0 will be provided once we are closer to release.
+Deis Workflow should use Kubernetes-native constructs wherever possible. By
+switching to `Deployments`, the controller component will no longer need to
+orchestrate rolling deploys. Instead, controller can delegate that work to
+Kubernetes control loops.
 
-Much of the Deis team's efforts are focused on Deis v2, which will see the Deis
-platform running on Kubernetes.
+* [X] Support Kubernetes Deployments <https://github.com/deis/controller/pull/854>
+* [ ] Kubernetes Deployments by default
 
-For details on Deis v2, see the [Deis v2 Design Document][].
+## Private Registry and Native IaaS Registry Support
 
-* [Alpha Issues](https://github.com/deis/deis/issues/4743)
-* [Beta Issues](https://github.com/deis/deis/issues/4809)
-* [Release Candidate Issues]()
+Many users of Deis v1 requested the ability to push and pull application
+artifacts from private Docker registries. While v1 supported a
+[workaround](https://github.com/deis/deis/issues/2232) or two, operators
+usually had to intervene or build custom tooling. Deis Workflow aims to make it
+simple to configure your Workflow install to pull and push images to private
+registries.
 
-[deis v2 design document]: https://github.com/deis/deis/issues/4582
-[v2]: https://github.com/deis/deis/labels/v2
+* [X] Pull from private registry <https://github.com/deis/workflow/pull/201>
+* [ ] Push to private registry for builder
+* [ ] Support native auth strategies for ECR and GCR
+
+## Application Management
+
+Various application-related features that give developers and operators
+flexibility in how applications are managed by the platform:
+
+* [ ] Per-process type health checks: <https://github.com/deis/controller/issues/881>
+* [ ] Application Maintenance Mode: <https://github.com/deis/deis/issues/3722>
+* [ ] Enforce SSL Per-Application: <https://github.com/deis/router/issues/148>
+* [ ] Per-application IP Whitelisting via CLI: <https://github.com/deis/controller/issues/240>
+* [ ] Per-application Kubernetes Network Policy:
+
+### Private Application Support
+
+* [ ] Private Application Support: <https://github.com/deis/controller/pull/934>
+
+Related issues:
+
+* <https://github.com/deis/deis/issues/4391>
+* <https://github.com/deis/deis/issues/2715>
+
+## Application Autoscaling
+
+Developers should be able to define an autoscaling policy per application
+process type. Under the covers, Workflow should use HorizontalPodAutoscaling.
+
+* [ ] Application Autoscaling <https://github.com/deis/workflow/issues/403>
+
+## Interactive `deis run /bin/bash`
+
+Provide the ability for developers to launch an interactive terminal session in
+their application environment.
+
+Related issues:
+
+* [ ] <https://github.com/deis/workflow-cli/issues/98>
```

Comment:
In some places "Related issues" are bullets... and in other places they're check boxes. I'd like to see a checkbox for the feature, and related issues be bullets... we might have lots of GH issues/PRs that are assembled for things like `deis run`